cpp/src/arrow/ArrowTestingConfig.cmake.in

```diff
@@ -34,3 +34,5 @@ include("${CMAKE_CURRENT_LIST_DIR}/ArrowTestingTargets.cmake")
 arrow_keep_backward_compatibility(ArrowTesting arrow_testing)
 
 check_required_components(ArrowTesting)
+
+arrow_show_details(arrow_testing)
```

Comment:
Ah, you expect `arrow_testing` not `ArrowTesting` here.
`arrow_testing` isn't a (CMake) package name. `ArrowTesting` is the (CMake) package name for this.
`ArrowTesting_FIND_QUIETLY` is defined but `arrow_testing_FIND_QUIETLY` isn't defined.